packages/contracts/src/lending/Lender.sol

```diff
@@ -168,7 +169,9 @@ abstract contract Lender is
         nonReentrant
     {
         // Checking whether the borrower has available funds for debt payment
-        require(_borrowerFreeAssets(msg.sender) >= debtPayment, "Not enough assets for payment");
+        if(_borrowerFreeAssets(msg.sender) < debtPayment) {
+            revert NotEnoughAssetsForPayment();
+        }
```

Comment:
Is it better practice? Or detected as vulnarability?

Reply:
Linter didn't allow it to pass and blocked the pipeline. But this change still makes sense because it makes code more cost-effective according to this discussion https://ethereum.stackexchange.com/questions/101782/requirecondition-message-vs-revert-with-a-custom-error-which-is-better-a